test/modules/appeals-service/AppealsService.test.ts

```diff
@@ -0,0 +1,233 @@
+                )
+                .reply(422);
+
+            try {
+                await appealsService.save(invalidAppeal as Appeal, BEARER_TOKEN);
+            } catch (err) {
+                expect(err.constructor.name).to.be.equal(AppealUnprocessableEntityError.name);
+                expect(err.message).to.contain(`save appeal on invalid appeal data`);
+            }
+        });
+
+        it('should return status 500 when internal server error', async () => {
+
+            nock(HOST)
+                .post(APPEALS_URI,
+                    JSON.stringify(appeal),
+                    {
+                        reqheaders: {
+                            authorization: 'Bearer ' + BEARER_TOKEN,
+                        },
+                    }
+                )
+                .reply(500);
+
+            try {
+                await appealsService.save(appeal as Appeal, BEARER_TOKEN);
+            } catch (err) {
+                expect(err.constructor.name).eq(AppealServiceError.name);
+                expect(err.message).to.include(`save appeal failed with message`);
+            }
+        });
+    });
+
+    describe('Loading appeals', () => {
+        it('should throw an error when arguments are not defined', () => {
+
+            const testAppealId = '123';
+            const testCompanyNumber = 'NI000000';
+
+            [undefined, null].forEach(async companyNumber => {
+                try {
+                    await appealsService.getAppeal(companyNumber!, testAppealId, BEARER_TOKEN);
+                } catch (err) {
+                    expect(err).to.be.instanceOf(Error)
+                        .and.to.haveOwnProperty('message').equal('Company number is missing');
+                }
+            });
+
+            [undefined, null].forEach(async appealId => {
+                try {
+                    await appealsService.getAppeal(testCompanyNumber, appealId!, BEARER_TOKEN);
+                } catch (err) {
+                    expect(err).to.be.instanceOf(Error)
+                        .and.to.haveOwnProperty('message').equal('Appeal id is missing');
+                }
+            });
+
+            [undefined, null].forEach(async invalidToken => {
+                try {
+                    await appealsService.getAppeal(testCompanyNumber, testAppealId, invalidToken!);
+                } catch (err) {
+                    expect(err).to.be.instanceOf(Error)
+                        .and.to.haveOwnProperty('message').equal('Token is missing');
+                }
+            });
+
+        });
+
+        it('should return an appeal when valid arguments are provided', async () => {
+
+            nock(HOST)
+                .get(`${APPEALS_URI}/${APPEAL_ID}`)
+                .reply(200, appeal);
+
+            const returnedAppeal = await appealsService
+                .getAppeal(appeal.penaltyIdentifier.companyNumber, APPEAL_ID, BEARER_TOKEN);
+
+            expect(returnedAppeal).to.deep.eq(appeal);
```

Comment:
May I suggest that in the future we use full form of equal method which is `.equal` which is bit more descriptive?